x-pack/plugins/cases/public/components/all_cases/utility_bar.tsx

```diff
@@ -56,47 +57,82 @@ export const CasesTableUtilityBar: FunctionComponent<Props> = React.memo(
 
     return (
       <>
-        <UtilityBar border>
-          <UtilityBarSection>
-            <UtilityBarGroup>
-              <UtilityBarText data-test-subj="case-table-case-count">
-                {i18n.SHOWING_CASES(totalCases)}
-              </UtilityBarText>
-            </UtilityBarGroup>
-            <UtilityBarGroup data-test-subj="case-table-utility-bar-actions">
+        <EuiFlexGroup
```

Comment:
I changed the code to use the `Eui` library and I removed the `UtilityBar` helpers.